mmsoap/client.py

```diff
@@ -1,14 +1,14 @@
 """
```

Comment:
oh sorry i got confused because your comment was on line 23 not the copyright part. I have already updated this to be proper comments